webui/tests/README.md

```diff
@@ -37,8 +37,30 @@ Before the tests can be started we need to build the cluster including the WebUI
 to make sure the served WebUI and cluster in general are up to date.
 
 Issue the following command:
-`./bin/e2e-tests.py e2e-tests` which in turn will:
+`./bin/e2e-tests.py e2e-tests` (or `make test`) which in turn will:
 
 1. Set up a test cluster
 2. Run the Cyrpess tests `Cypress run`
 3. Tear down the cluster and clean up in case of errors
+
+## Test Development
+
+Use `make dev-tests` to set up for test development and then proceed to add new
+tests suites or update and rebuild the WebUI artifacts to see changes in tests.
+
+### Debugging Test Issues
+
+For reproducing and catching test flakes there is a simple helper script `./bin/try-for-flake.sh`.
+Just executing the script will run `make test` over and over until it hits a error.
+
+To speed up this process you can:
+
+1. try to avoid the whole cluster set up and tear down on each iteration
+2. limit the test scope:
+  - use `.skip` on each unwanted test suite or `.only` on the target suite.
```

Comment:
question: does this actually work with our cypress setup? It looks like it's a mix bag. I tried applying `.only` to the very first test in `02-loginout.spec.ts` and it ran all the other test files fully, and when it came to `02-loginout.spec.ts` it did run just the first test where I applied `.only`... Is that the correct behavior or is there a weird setup going on?

Reply:
update: here's what worked for blocking full test files. If you apply `x` to the top level `describe` (`xdescribe`) it stops the entire `describe` block in that file from running. So to isolate the tests I want to run, I need to add `x` to all the other files.

Reply:
it was mixed bag last time I used it. from my googling, it's about the Jest version that Cypress is using. It is supposed to work so I think it's safe to have it in the readme. I usually just delete the unwanted test suites.
https://github.com/cypress-io/cypress/issues/5441
https://github.com/cypress-io/cypress/issues/632
https://github.com/cypress-io/cypress/issues/5345